client/src/cmdhficlass.c

```diff
@@ -1222,6 +1222,52 @@ static int CmdHFiClassEView(const char *Cmd) {
     return PM3_SUCCESS;
 }
 
+static int CmdHFiClassESetBlk(const char *Cmd) {
+    CLIParserContext *ctx;
+    CLIParserInit(&ctx, "hf iclass esetblk",
+                  "Display emulator memory.\n"
```

Comment:
I believe you need to update this texts to match the new command

Reply:
Whoops missed that one!